spec/features/access_start_page_spec.rb

```diff
@@ -1,10 +1,21 @@
 require 'rails_helper'
 
-RSpec.feature "Access Start Page" do
-  scenario "user will read text and begin form" do
+RSpec.feature "Access Start Page", js: true do
+  let(:respondents_details_page) { ET3::Test::RespondentsDetailsPage.new }
+
+  scenario "user will be able to read text" do
     start_page.load
 
     expect(start_page).to be_displayed
-    expect(start_page).to have_hello_world
+    expect(start_page).to have_header
```

Comment:
Im not sure how much value there is in testing that the page just has a '.content-header' etc..

If we were doing multiple languages, I would say we should match by the text in the page object, so when you say 'have_header' - it means 'Have the header with the correct text'

So, you can either switch to doing that now, expecting a welsh version in the future, or limit the access start page spec to just validating that it got there (to be_displayed).  I know it doesnt mean much, but it proves the page is there.  Normally we dont go checking static content, but of course when we switch to multiple languages, all of a sudden the text is no longer static but dynamic, therefore justifying validating it.